src/main/java/roboy/dialog/states/ordinaryStates/IntroductionState.java

```diff
@@ -227,10 +196,10 @@ private String getRoboyFactsPhrase(Roboy roboy) {
         MemoryNodeModel node = getMemNodesByIds(roboy.getRelationships(predicate)).getRandomElement();
         if (node != null) {
             String nodeName = "";
-            if (node.getProperties().containsKey("full_name") && !node.getProperties().get("full_name").equals("")) {
-                nodeName = node.getProperties().get("full_name").toString();
+            if (node.getProperties().containsKey(full_name) && !node.getProperties().get(full_name).equals("")) {
```

Comment:
So, you propose to encapsulate? I think it makes sense to do for Roboy since we know what this is about, however, for a random MemoryNodeModel, I am not sure